tools/win32/detect_cpu_extensions_wine.py

```diff
@@ -68,14 +69,12 @@ def path_as_windows(fpath):
 
 def write_summary(allcodes):
     """Write a summary of all found codes to stdout."""
-    print """\n
-----------------------------------------------------------------------------
-Checked all binary files for CPU extension codes. Found the following codes:"""
+    print("\n"
+          "----------------------------------------------------------------------------"
+          "Checked all binary files for CPU extension codes. Found the following codes:")
```

Comment:
Last time I checked, the release scripts had some py2-isms, so that the release process only worked on python 2.x.

Anyway, I suggest focussing this PR on the docstrings which are actually user-facing, and looking into removing dead code from `tools/` in a separate PR (which will need to be vetted by Ralf or Pauli).

BTW, the bulk of the changes here LGTM.